cordformation/src/test/kotlin/net/corda/plugins/DockerformTest.kt

```diff
@@ -3,24 +3,23 @@ package net.corda.plugins
 import com.typesafe.config.ConfigFactory
 import org.assertj.core.api.Assertions.assertThat
 import org.gradle.testkit.runner.TaskOutcome
+import org.gradle.testkit.runner.UnexpectedBuildFailure
 import org.junit.jupiter.api.Test
 import kotlin.test.assertEquals
+import kotlin.test.assertFailsWith
 import kotlin.test.assertTrue
 
 class DockerformTest : BaseformTest() {
 
     @Test
-    fun `deploy a node with cordapp dependency`() {
+    fun `fail to deploy a node with cordapp dependency with missing dockerImage tag`() {
         val runner = getStandardGradleRunnerFor(
                 "DeploySingleNodeWithCordappWithDocker.gradle",
                 "prepareDockerNodes")
 
-        val result = runner.build()
```

Comment:
I like the extra test. Gradle's test-kit also allows you to test error paths like this:
```
val result = runner.buildAndFail()
```
This allows you to inspect the outcome of the `prepareDockerNodes` task so you can be _sure_ that the test failed for the correct reason.